_posts/2018-11-01-Inter-service-communication.md

```diff
@@ -0,0 +1,327 @@
+---
+layout: post
+authors: [kevin_van_houtte]
+title: 'Communication in a distributed system with OpenFeign: Tips & Tricks'
+image: /img/intercommunication/intercomm_header.jpg
+tags: [Microservices, Security, Spring, Cloud, OpenFeign]
+category: Microservices
+comments: true
+---
+
+In contrast to monolithic applications, services in a distributed system are running on multiple machines. 
+To let these services interact with each other, we need some kind of inter-process communication mechanism.
+With the help of OpenFeign, I will explain how we can fire off synchronous calls to another service.
+
+
+# Table of contents
+1. [Setup](#setup)
+2. [Different kind of HTTP Clients](#different-kind-of-clients) 
+3. [Enabling Mutual SSL](#building-a-mutual-ssl-client-with-apache)
```

Comment:
Title needs to be updated to "Building a Mutual SSL client with Apache"?
```suggestion
3. [Building a Mutual SSL client with Apache](#building-a-mutual-ssl-client-with-apache)
```

Reply:
Or he could keep the title "Enabling Mutual SSL" and explain how to do it with the Apache HTTP client as an example since most if not all clients support mutual SSL.
This could cause some confusion for the reader imo.

Reply:
Fair enough. :)